tests/integration/local/invoke/test_integrations_cli.py

```diff
@@ -291,6 +291,27 @@ def test_invoke_returns_expected_result_when_no_event_given(self):
         self.assertEqual(process.returncode, 0)
         self.assertEqual("{}", process_stdout.decode("utf-8"))
 
+    # @pytest.mark.flaky(reruns=3)
```

Comment:
nit: left over from debugging I assume

Reply:
Good catch. It made tests runs faster but forgot to adjust.